tests/jest/setup.js

```diff
@@ -1,24 +0,0 @@
-// `mongodb-memory-server` downloads a binary on first run in CI, which can take
-// a while, so we bump up the timeout here.
-jest.setTimeout(60000); // in milliseconds
-
-// AVA-like test wrapper for known failing tests
-// see: https://github.com/avajs/ava#failing-tests
-test.failing = (title, testFn) => {
```

Comment:
wasn't used by any thing in our codebase as far as search was concerned, happy to add this back in if anyone else has contrary context.